### PR TITLE
withdraw request for oldest first bug fixed

### PR DESCRIPTION
**Pull request by shruthiklk15**
* Under the guidance of @code-sumedh I have fixed this bug, please review and merge.

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -3,37 +3,55 @@ const connectionUrl =
...
-        var elem = document.querySelector("[data-test-dialog-primary-btn]");
-        console.log(elem);
-        elem.click();
-        startCounter = olderFirst ? startCounter - 1 : startCounter + 1;
-        counter++;
-        syncIt();
-      }, 700);
-    };
-    syncIt();
+    goToLastPage(olderFirst);
```
Comment (ishwarrimal) [inline scripts/index.js]: let's have the check here instead of in the function.
if(olderFirst){
goToLastPage()
}

File: scripts/index.js [hunk 2]
```diff
@@ -3,37 +3,55 @@ const connectionUrl =
...
+            }
+          }, 700);
+        }
+        
+      };
+      syncIt();
+    }, 2000);
+  }
+
+  function goToLastPage(olderFirst) {
```
Comment (ishwarrimal) [inline scripts/index.js]: if you follow the condition I suggested above, we don't need to pass parameter.

File: scripts/index.js [hunk 3]
```diff
@@ -3,37 +3,55 @@ const connectionUrl =
...
+    if (olderFirst) {
```
Comment (ishwarrimal) [inline scripts/index.js]: no need for condition as well

File: scripts/index.js (hunk 1 above)
Comment (shruthiklk15) [reply scripts/index.js]: okay

File: scripts/index.js (hunk 2 above)
Comment (shruthiklk15) [reply scripts/index.js]: ok

File: scripts/index.js (hunk 3 above)
Comment (shruthiklk15) [reply scripts/index.js]: ok

**Review by ishwarrimal — APPROVED** [review]

**Pull request merged**